### DISCUSSION TITLE
How to clear recent repositories?

### DISCUSSION THREAD

**Discussion by myungjinki** [community/community#34416] [Other Feature Feedback, Questions, & Ideas]
I searched google, stackoverflow, but I couldn't find the way to solve it.

And I tried lot's of thins to do

Is there a way to clear recent repositories?

**Comment by fadyat** [comment]
I think GitHub doesn't support that option.

Meme:
You can delete the repository 😄 
But this trick doesn't work with repositories other than yours 😢
